include/ppx/geometry.h

```diff
@@ -261,6 +263,18 @@ class Geometry
     void AppendTriangle(const TriMeshVertexData& vtx0, const TriMeshVertexData& vtx1, const TriMeshVertexData& vtx2);
     void AppendEdge(const WireMeshVertexData& vtx0, const WireMeshVertexData& vtx1);
 
+    // Use the following resize functions before using the append functions for speed over convenience,
+    // corresponding to Buffer "Use #1" described above.
+    // Caller is responsible for resizing Buffers to the appropriate size.
+    //
+    // Enabling the initial resize mode will indicate that initial resizing is expected,
+    // and it will change the behaviour of all the Append functions used afterwards to use
+    // the Buffer DataPtrs.
+    //
+    void EnableInitialResizeMode();
```

Comment:
Is there actually a use case where an application wants to switch the resize mode of a geometry object?

If there isn't, this should come in via the create info or options. It's not advisable to have operational states be changeable if they cause objects to switch personalities.